### ISSUE TITLE
Send message to single socket.Id()?

### ISSUE THREAD

**Issue by Xeoncross** [googollee/go-socket.io#132]
On the server, I have the `socket.Id()` as a string from a client. I want to forward this message to that given `socket` connection. How can I do this? I from the global server instance I only see [BroadcastTo()](https://github.com/googollee/go-socket.io/blob/master/server.go#L91).

In socket.io for node.js I could lookup the socket like so:

```
// sending to individual socketid
socket.broadcast.to(socketid).emit('message', 'for your eyes only');
```

**Comment by Xeoncross** [comment]
Thinking about it some more, the javascript version has all clients auto-join a room matching their `session.Id()`. I couldn't find anything about this library doing the same so perhaps I just need to add an `so.Join(so.Id())`.

**Comment by Equim-chan** [comment]
Same problem here. I also used `so.Join(so.Id())` to resolve it but it doesn't seem like a good solution.

**Comment by LionelWoody** [comment]
same problem.

**Comment by vincentblockchain** [comment]
same problem

**Comment by behsh2000** [comment]
same problem.

but mabe we can use socket id as  a room.
server.BroadcastTo("roomname", "chat message", msg)

**Comment by mrfoe7** [comment]
@Xeoncross i implemented this method in PR #277
